### ISSUE TITLE
past p3

### ISSUE THREAD

**Issue by hectoralos** [apertium/apertium-chv#13]
There are problems in p3.sg and p3.pl in the past tense.

According to Krueger (p. 144) ч appears in p3 of stems in /l n r/ (actually he says p3.sg but in the examples he gives we can see that it also happens in p3.pl). So:

```
$ echo "кай<v><iv><past><p3><sg>" | hfst-lookup .deps/chv.LR.lexc.hfst 2>/dev/null
кай<v><iv><past><p3><sg>	кай>{T}>ӗ	0,000000
```
Currently кайчӗ is generated instead of кайрӗ
```
$ echo "кай<v><iv><past><p3><pl>" | hfst-lookup .deps/chv.LR.lexc.hfst 2>/dev/null
кай<v><iv><past><p3><pl>	кай>{T}>ӗҫ	0,000000
```
Currently кайчӗҫ is generated instead of кайрӗҫ

(By the way, https://www.sapatlav.club is useful for conjugating verbs in Chuvash)

**Comment by ftyers** [comment]
Hopefully fixed in 954f27e 

```
кай<v><iv><past><p1><sg>	кайрӑм	0,000000
кай<v><iv><past><p2><sg>	кайрӑн	0,000000
кай<v><iv><past><p3><sg>	кайрӗ	0,000000
кай<v><iv><past><p1><pl>	кайрӑмӑр	0,000000
кай<v><iv><past><p2><pl>	кайрӑр	0,000000
кай<v><iv><past><p3><pl>	кайрӗҫ	0,000000
```

Re: sapatlav, ... perhaps we could scrape it and run whole lot through the analyser at the same time ?

**Comment by hectoralos** [comment]
Works.
About sapatlav: what I've been doing is extracting all forms of the verbs кил, кай and ту and run our analyser. Now all forms are recognized. But yesterday I "discovered" two new tenses and a few new gerunds (which are all explained in Sergeev's thick book written in Chuvash). I'll try to introduce everything this week end. You will find new issues because of them.
